### DISCUSSION TITLE
Unable to login to ArcGIS Enterprise, shows "Invalid redirect_uri"

### DISCUSSION THREAD

**Discussion by patelmanya** [EsriPS/arcgis-assistant-feedback#33] [Q&A]
The app has been registered with the portal and using the App ID shows Invalid redirect_uri. 

Getting same error with 10.8.1 & 10.6.1 versions.

Am I missing something?

![image](https://user-images.githubusercontent.com/1124851/140676729-27ea0636-6ace-4dd3-a513-38fa86dcd52a.png)

**Comment by jpeterson** [comment]
Hi @patelmanya, just to confirm could you share the Redirect URI you have set for the item in your Enterprise portal?

![image](https://user-images.githubusercontent.com/3399064/141025202-31e3e2df-304d-48d4-aded-35841951d104.png)

**Reply by clamontca** [reply]
Hi, I'm getting the 'portal is not reachable' error as well. Is there a solution? The portal is running 10.6.1.

**Comment by eaustin1** [comment]
Hello - was anyone else able to solve this? I have the correct Redirect URI - was able to sign into my Portal 10.6
But Assistant is giving me: 
![image](https://user-images.githubusercontent.com/94077420/141196116-c5c32537-e8a7-4f5b-9578-9f2c7da28ec7.png)


When I hit the given URL - it seems to be functional:
![image](https://user-images.githubusercontent.com/94077420/141196187-b7829d63-7120-431a-ad20-d785316de26e.png)

**Reply by jpeterson** [reply]
Hi @eaustin1. Can you confirm the URL you used while signing in to ArcGIS Enterprise (in the screen below)? 

I want to confirm it is `https://gis.atwell-group.com/arcgis` and not `https://gis.atwell-group.com/arcgis/sharing/rest`

![image](https://user-images.githubusercontent.com/3399064/141204046-b74a75e5-22db-47e2-9ee8-0fa6383f222d.png)

**Reply by eaustin1** [reply]
@jpeterson - Yes I signed in without the rest:
![image](https://user-images.githubusercontent.com/94077420/141314540-b5db0cee-1b0d-4818-94d5-f71268484f36.png)


But it takes me back to the rest:
![image](https://user-images.githubusercontent.com/94077420/141314577-9f8113e3-a8bf-4a3d-9daf-d7c08475cd99.png)

**Reply by jpeterson** [reply]
@eaustin1 this is very curious (and a tough one to reproduce). Would you be open to a quick 5-10 minute screensharing session sometime so we can better understand what is going on? If so, shoot me an email (jpeterson@esri.com) and we can go from there.

Alternatively, we can start digging deeper in this thread – starting with exploring any console errors you are seeing when arriving back in Assistant after the OAuth workflow has been completed.

**Reply by wboras** [reply]
I am having the same problem: Your portal is currently not reachable. Were you able to find a solution?
